src/styles/blocks/page.scss

```diff
@@ -0,0 +1,101 @@
+body {
+  @include main-font-set;
+
+  margin: 0;
+}
+
+.page-cont {
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
```

Comment:
move this rule to `*` selector

Reply:
https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules

Don't use * selector (it impacts performance)
Set styles only for elements that require them. Zeroing out your margins, paddings or other styles with '*' is still inefficient for browser.

Should I still do it?

Reply:
yes, for  box-sizing: border-box; its ok